### DISCUSSION TITLE
ERROR > Failed to archive ios/Runner.xcworkspace

### DISCUSSION THREAD

**Discussion by iamSahdeep** [codemagic-ci-cd/codemagic-docs#1655] [Q&A] [answered]
Hi our workflow is giving this error only for our one client (We use env variables to use single workflow but different details to build app  for our clients )
build : https://codemagic.io/app/5e5c268ce389931b59c2fc41/build/6461ca931db40366835778bc

```
[06:35:03] ERROR > Failed to archive ios/Runner.xcworkspace


Build failed :|
Step 18 script `Building IPA` exited with status code 65
```

It only fails on this step for this with this clients details only. we have several other clients data as well and they work perfectly. for example this : https://codemagic.io/app/5e5c268ce389931b59c2fc41/build/6461caa6227f5af5a4318786

There are no mentioned error and we could not find anything related. Not sure if anything is missing cause all the profiles and certificated are created successfully in above steps.

**Comment by icarusdust** [comment, marked as answer]
Hi @iamSahdeep, I am reading the following error:

`error: Provisioning profile "SSPConnects xxxxxxx" doesn't support the Communication Notifications capability. (in target 'Runner' from project 'Runner')`

In order to get it resolved, you will need to enable the Communication Notifications capability in your Apple Developer account >> Identifiers

**Reply by iamSahdeep** [reply]
Thanks @icarusdust Let me verify this. 
Just one more thing, where did you get this log from. I mean is it internal? Cause i did not see it in the build.
Usually if this is the case, i am getting the exception in script and the command highlights the issue clearly but in this build i did not get anything.

**Reply by icarusdust** [reply]
You can see more of the Xcode logs if you can add `- /tmp/xcodebuild_logs/*.log` to the artefacts section in codemagic.yaml and that will generate a zip file that contains all the necessary logs. 

On a different note, I see that your team is on a paid plan which means that you have access to the private chat and that it is a paid feature. You can use the chat widget in the UI for quicker responses as our team prioritises messages there.

**Reply by iamSahdeep** [reply]
Thanks @icarusdust Will be using chat for any issues. Thanks
